CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+## [1.4.0]
```

Comment:
However, we apparently have to bump the version in the main.
```./tools/bump-version.sh 1.5.0```

Reply:
Dropped commit.

Reply:
Still think we can add an entry for the underscore under unreleased.

Reply:
But then the underscore in not a part of 1.4, until we back port that commit to 1.4 branch. The underscore change itself is not worth entry in changelog until it is meant to be in 1.5 release. But we can not add it to 1.5, because we don't have 1.4 yet.

Reply:
Added note to the CHANGELOG.